yb-voyager/cmd/exportSchema_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/stretchr/testify/assert"
+	"strings"
+	"testing"
+)
+
+func TestShardingRecommendations(t *testing.T) {
```

Comment:
suggestions for more unit tests:
1. MVIEW with `WITH` clause having other storage parameters than `COLOCATION = true/false`

2.  ALTER TABLE stmts - since table.sql can have that also.

Reply:
@hbhanawat, YB might not be supporting any other storage parameters.
But the DDL can still have it, because of the source(PG) supporting that - [link](https://www.postgresql.org/docs/current/sql-creatematerializedview.html#:~:text=for%20more%20information.-,WITH%20(%20storage_parameter%20%5B%3D%20value%5D%20%5B%2C%20...%20%5D%20),-This%20clause%20specifies)